src/test/java/de/bonndan/nivio/config/NivioConfigPropertiesTest.java

```diff
@@ -83,4 +83,16 @@ void checksLogoUrl() {
         ConstraintViolation<NivioConfigProperties>[] constraintViolations = (ConstraintViolation<NivioConfigProperties>[]) validate.toArray(ConstraintViolation[]::new);
         assertThat(constraintViolations[0].getMessage()).isEqualTo("brandingLogoUrl must be a valid URL");
     }
+
+    @Test
```

Comment:
👍 

You could write a positive test, too. (Although they are missing everywhere).

Reply:
okay, in the future, i will try to write a positive test and to write the commits in english